pl_bolts/datamodules/cityscapes_datamodule.py

```diff
@@ -82,13 +86,18 @@ def __init__(
                 'You want to use CityScapes dataset loaded from `torchvision` which is not installed yet.'
             )
 
-        self.dims = (3, 32, 32)
+        if target_type not in ['instance', 'semantic']:
+            raise ValueError(f'Only "semantic" and "instance" target types are supported. Got {target_type}.')
+
+        self.dims = (3, 1024, 2048)
         self.DATASET = Cityscapes
```

Comment:
This shall be class constant

Reply:
Making it a class constant resulted in failing doctest (unwrap loop). I removed it and just use dataset class imported from torchvision, same as in other datamodules.